README.md

```diff
@@ -166,7 +169,25 @@ TEXTTOSPEECH_USER=<add tts username>
 TEXTTOSPEECH_PASSWORD=<add tts password>
 ```
 
-### 5. Run the application
+### 5. OPTIONAL Create a virtual environment
+
+$ `virtualenv watson-voice-bot`
```

Comment:
For someone new to virtualenv, they're going to stumble here, A) not knowing what `virtualenv` is without prior introduction, B) not having installed `virtualenv` or been linked to relevant documentation as part of the pattern, and C) not knowing that they shouldn't run `deactivate` immediately after activating the environment as documented below.